README.md

```diff
@@ -28,8 +28,8 @@ you do not need to do anything else other than unzip and double click on **tvb_s
 
 **NOTE:**
 
-To contribute to this repo, you are advised to first fork it under GitHub (through the web ui), 
-and next clone that fork on your machine, and use it as you use any other git repository.
+To contribute to this repo, you are advised to first fork it under GitHub (through the web UI), 
+and next clone that forks on your machine, and use it as you use any other git repository.
```

Comment:
Sorry, this is wrong, the fork is singular noun here, not a third person conjugation of the verb fork.